packages/eth-explorer/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@obsidians/eth-network": "^0.3.0",
+    "@obsidians/eth-sdk": "^0.3.0",
```

Comment:
I didn't see any reference from `@obsidians/eth-sdk`, what's the reason for adding the dependence

Reply:
> I didn't see any reference from `@obsidians/eth-sdk`, what's the reason for adding the dependence

Because there is a reference to a dependency here but no dependency is introduced 
 [https://github.com/ObsidianLabs/eth-components/blob/master/packages/eth-explorer/src/AccountExplorer.js#L17](url)